governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md

```diff
@@ -0,0 +1,58 @@
+---
+title: SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - June 17, 2024
```

Comment:
```suggestion
title: SparkLend Ethereum - Update Morpho DAI Vault Supply Caps - June 17, 2024
```

Shouldn't this title match the file name and the title below?

Reply:
yes, we will update this

Reply:
https://github.com/makerdao/community/pull/2671